pkg/controller/velerobackup/velerobackup_controller.go

```diff
@@ -33,6 +35,9 @@ import (
 const (
 	controllerName = "velerobackup"
 	errChecksum    = "HIVE_CHECKSUM_ERR_97A29D08"
+
+	// Default reconcile rate limiting to 1 backup every 3 minutes.
+	defaultReconcileRateLimitDuration = 180 * time.Second
```

Comment:
Small nit. Sorry that I didn't see this before. But this is more readily if it uses `3 * time.Minute` since the comment is referring to minutes. This does not need to be done for this PR, though.